src/main/java/com/hackoeur/jglm/Mat3.java

```diff
@@ -231,16 +231,16 @@ public int getNumColumns() {
 	
 	@SuppressWarnings("unchecked")
 	@Override
-	public <T extends Vec> T getColumn(final int columnIndex) {
```

Comment:
Here a hidden cast will happen, which you'll have to do explicitly anyway if the compiler can not work out which vector type you're assigning to.